### DISCUSSION TITLE
Dropdown Menus with Titanium Theme in Rich Text Documents are Transparent

### DISCUSSION THREAD

**Discussion by mystic-githuber** [standardnotes/forum#2882] [General]
**Describe the bug**
Dropdown menus are transparent in the rich text document editor with the titanium theme.

![image](https://user-images.githubusercontent.com/66426048/197533681-b515e225-3818-4cdd-b4d1-d23167275fdf.png)

**Comment by effieeee** [comment]
Took note of a similar report of this over email, @mystic-githuber may I know if it remains transparent on the Titanium theme, even after switching to a different theme and returning to Titanium?

**Comment by mystic-githuber** [comment]
@effieeee 

Yes, that seems to be the case even after switching themes.

**Comment by effieeee** [comment]
Thanks! I'll take note that it happens the same way for you when I report the issue!
